frontend/components/user/UserApproveCard.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+  <div
+    class="user-main-card"
+    @mouseover.prevent="handleMouseOver"
+    @mouseleave="handleMouseLeave"
+  >
+    <div class="user-card" :class="{ 'user-card-white': variant == 'white' }">
+      <div class="user-card-header">
+        <div
+          class="user-icon-container"
+          :style="'background-color:' + color"
+          :class="{ 'icon-sm': icon === 'small' }"
+        >
+          <img src="@/assets/images/user_icon.svg" alt="community" />
+        </div>
+      </div>
+      <div class="user-card-body">
+        <div>
+          <h5 class="field-names">
+            {{ getName() }}
+          </h5>
+          <h5 class="field-kinds">
+            {{ verify.community.name }}
+          </h5>
+        </div>
+      </div>
+    </div>
+
+    <div class="user-btn-container">
+      <b-button
+        variant="dark"
+        block
+        size="sm"
+        @click="
+          handleUser($event, verify, {
+            verify: 'verify'
+          })
+        "
+        >Verify</b-button
+      >
+      <Reject :id="verify.id" type="community" :member="verify"></Reject>
+    </div>
+  </div>
+</template>
+
+<script>
+import Reject from '@/components/RejectModal.vue'
+import { getApiUrl, getCookie } from '@/plugins/utils.js'
+
+export default {
+  components: {
+    Reject
+  },
+  props: {
+    verify: {
+      type: Object,
+      default: () => {
+        return {}
+      }
+    },
+    color: {
+      type: String,
+      default: 'RGB(255, 255, 255)'
+    },
+    go: {
+      type: Boolean,
+      default: true
+    },
+    variant: {
+      type: String,
+      default: 'normal'
+    },
+    icon: {
+      default: 'large',
+      type: String
+    }
+  },
+  data() {
+    return {
+      hover: false
+    }
+  },
+  methods: {
+    getName() {
+      const user = this.verify.user
+      if (user.first_name === '' || user.last_name === '') {
+        return user.username
+      } else {
+        return `${user.first_name} ${user.last_name}`
+      }
+    },
+    async handleUser(e, tv, { verify, reject }) {
```

Comment:
What does this function do? (add a comment)

/**
 * handleUser(e, dv, {verify, reject})
 * e - describe this arg.
...
 */

Reply:
It handles verifying users, though it's poorly named, in my opinion. @bomiku I know this isn't your code, but you can consider improving it since you've bumped into it.

Reply:
I just changed the function name to "submitUserVerification" for a clearer name, comment no longer needed. Thanks!